examples/jsm/interactive/HTMLMesh.js

```diff
@@ -371,7 +374,8 @@ function html2canvas( element ) {
 
 				color.set( accentColor );
 
-				const luminance = Math.sqrt( 0.299 * ( color.r ** 2 ) + 0.587 * ( color.g ** 2 ) + 0.114 * ( color.b ** 2 ) );
+				ColorManagement.getLuminanceCoefficients( lumaCoeffs );
+				const luminance = Math.sqrt( lumaCoeffs.x * ( color.r ** 2 ) + lumaCoeffs.y * ( color.g ** 2 ) + lumaCoeffs.z * ( color.b ** 2 ) );
```

Comment:
Do you have any idea what this is computing? I'm not sure you should worry about this code chunk...

Reply:
Asking for clarification in https://github.com/mrdoob/three.js/pull/24030/files#r1679662069, but I'd be OK with skipping this file too.

Reply:
Let's skip HTMLMesh.js for now, as this chunk of code is otherwise unrelated to the rendering pipeline.